addons/api/addon/generated/models/account.js

```diff
@@ -13,13 +13,11 @@ export default class GeneratedAccountModel extends BaseModel {
   // =attributes
 
   @attr('string', {
-    readOnly: true,
```

Comment:
Are these readonly fields or not? It's not clear to me in this PR since it seems like they're not supposed to be able to be updated but now we're manually deleting them in the serializer? What's the actual bug?

Reply:
@ZedLi the original bug was not allowing the name and the description of an account to be update.  We set the fields to readOnly and although this works for updating the name and description of the account.  It was causing a he creation of a new account to fail because it wanted the auth_method_id and the type.  This is why we are deleting the fields in the serializer if it's not new